WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingAddPackageViewModel.swift

```diff
@@ -212,7 +212,16 @@ final class WooShippingAddPackageViewModel: ObservableObject {
             self.selectedSavedPackageId = nil
         }
 
-        return nil
+        let deleteAction = WooShippingAction.deletePackage(siteID: siteID, packageID: packageToRemove.id) { result in
+            switch result {
+            case .success(let response):
+                self.transformSavedPackages(response)
+            case .failure:
+                break
```

Comment:
Could we log this error to help with troubleshooting in case the request fails?

Also, since we optimistically remove the package from the local list, how about we undo that if the request fails?

Reply:
Yeah, I was planning to do that, but forgot to. As you can see in `starUnstarPackage` I do the undo if it fails, and even added TODO to check if that is ok :D